src/styles/style.css

```diff
@@ -141,13 +112,20 @@ h6 {
   margin-bottom: 0.5em;
 }
 
+/* general */
+
+li:hover,
+.join-btn:hover,
+.social-icon:hover {
+  transform: scale(1.03) !important;
```

Comment:
Great idea, I think this works really well!

I'd suggest we avoid `!important` wherever possible :sweat_smile:
It seems to work without it, I think?

At the moment the social icons expand downwards and the day/night toggle expands upwards, I think we should fix that before we merge this.

Maybe something like:
```
.social-container {
  display: flex;
  align-items: center;
  justify-content: center;
}
```
and stick a `social-container` class on the <a>'s that contain the icons? I think we could lose the padding around the social icons then as well :sparkles:

Reply:
Thanks for the drafting/testing `!important` tag catch!

For the icons, I'm a bit confused, but are you open to make that change and push it so I see what you mean?

(So you're not alarmed, I was also just playing around with the join button hover too and having that one change gradiant instead of zoom.)

Reply:
I've pushed it up, didn't need the extra class in the end!

I removed the padding around the footer for now as we weren't really using it, WDYT?

Reply:
Looks great! Thanks!